static/js/components/tiles/map_tile.tsx

```diff
@@ -389,6 +390,19 @@ export function draw(
     height,
     chartData.geoJson
   );
+  // Re-fit projection to border data if border data is available.
+  // This prevents borders from getting cutoff if enclosed places don't
+  // stretch wall-to-wall.
+  if (!_.isEmpty(chartData.borderGeoJson)) {
```

Comment:
instead of re-fitting the projection, could just use the borderGeoJson to get the projection in the first place?